account_invoice_kanban/models/account_invoice.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class AccountInvoice(models.Model):
+    _name = 'account.invoice'
+    _inherit = ['account.invoice', 'base.kanban.abstract']
+    _order = 'date_due asc'
+
+    @api.multi
+    def _read_group_stage_ids(self, stages, domain, order):
```

Comment:
This is implemented [in the base](https://github.com/OCA/server-tools/blob/10.0/base_kanban_stage/models/base_kanban_abstract.py#L106) & doesn't look to be changed here. I think you can delete

Reply:
@nikul-serpentcs I agree with @lasley.

You inherit `base.kanban.abstract` and here the original code:
```
    @api.multi
    def _read_group_stage_ids(self, stages, domain, order):
        search_domain = [('res_model_id.model', '=', self._name)]
        if domain:
            search_domain.extend(domain)
        return stages.search(search_domain, order=order)
```
Then here:
```
    @api.multi
    def _read_group_stage_ids(self, stages, domain, order):
        search_domain = [('res_model_id.model', '=', self._name)]
        return stages.search(search_domain, order=order)
```

So the only real change is you remove the if condition, may I ask why?

Reply:
@victormartinelicocorp Yes the code was changed a long time ago, you are referring to the old code.

Please check the latest changes where the @lasley comment is attended, thanks!

Reply:
@darshan-serpent Could you help me with the review and give the link to the comment on @lasley you mention?

About the code
I am looking at this: https://github.com/OCA/account-invoicing/pull/309/files#diff-f0c2828b9fe5cf1e5f1cd28f2a52f4c2
And this: https://github.com/OCA/server-tools/blob/10.0/base_kanban_stage/models/base_kanban_abstract.py#L109

I am not necessary subscribing all opinions of others reviewers so please attend my concern that is:

- Here you are overwriting a function that seems not been changed just for an if condition that perhaps is necessary.
I am not asking for change the code if you give a reason.

Reply:
This can be removed now due to https://github.com/OCA/server-tools/pull/989